lib/components/viewers/trip-viewer.js

```diff
@@ -39,19 +40,15 @@ const RouteName = styled.h2`
 const HeaderText = styled.h1`
   margin: 2px 0 0 0;
 `
-
-const TripDescription = styled.div`
-  background-color: rgba(19, 193, 193, 0.2);
-  border-radius: 5px;
-  box-shadow: 2px 2px 5px 1px rgba(0, 0, 0, 0.02);
-  font-size: 12px;
-  margin-bottom: 1em;
-  padding: 0.75em 1em;
-`
 const FlexWrapper = styled.div`
   display: flex;
   width: ${(props) => props.width};
 `
+const InvisibleAccessibilityLabel = styled.span`
+  height: 0;
+  overflow: 0;
+  width: 0;
```

Comment:
For spans, you need `display: block` or `display: inline-block` for the text to be clipped.

Reply:
The overflow property is wrong too, what a mess, thank you!!